pkg/tcpip/header/icmpv4.go

```diff
@@ -150,6 +156,12 @@ func (b ICMPv4) SetSequence(sequence uint16) {
 	binary.BigEndian.PutUint16(b[icmpv4SequenceOffset:], sequence)
 }
 
+// Set Time for Timestamp reply messages
+func (b ICMPv4) SetRxTimeStamp(t_rx int64,t_tx int64)  {
+    binary.BigEndian.PutUint32(b[12:], uint32(t_rx))
+    binary.BigEndian.PutUint32(b[16:],uint32( t_tx))
```

Comment:
Note minor spacing issue. You should be able to fix all spacing issues by using gofmt on the file. (Many editors can do use on save.)